### PR TITLE
Handle promises correctly

### PR DESCRIPTION
**Pull request by paulcbetts**
It looks like the last part of the promisification bits have some issues, this fixes it up

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,14 +1,13 @@
-'use strict';
```
Comment (sindresorhus) [inline index.js]: Don't do unrelated changes.

File: index.js [hunk 2]
```diff
@@ -1,14 +1,13 @@
...
 const pify = require('pify');
+const getos = pify(require('getos'));
 
 module.exports = () => {
 	if (process.platform !== 'linux') {
 		return Promise.reject(new Error('Only Linux systems are supported'));
 	}
 
-	return execa('lsb_release', ['-a', '--short']).then(res => {
+	return execa('lsb_release', ['-a', '--short']).then((res) => {
```
Comment (sindresorhus) [inline index.js]: `(res)` => `res`

File: index.js (hunk 1 above)
Comment (paulcbetts) [inline index.js]: hwhoops, sorry

File: index.js (hunk 2 above)
Comment (paulcbetts) [inline index.js]: Is that legal ES6?

Comment (SamVerschueren) [inline index.js]: Yes :)

Comment (paulcbetts) [inline index.js]: :+1:

**Comment by kevva** [comment]
Thanks :)!
